janus_core/calculations/md.py

```diff
@@ -159,6 +159,12 @@ class MolecularDynamics(BaseCalculation):
         Keyword arguments to control post-processing operations. Default is None.
     correlation_kwargs
         Keyword arguments to control on-the-fly correlations. Default is None.
+    plumed_input
+        PLUMED input or path to a PLUMED input file. If provided,
+        the ASE Plumed calculator will be used to wrap the MLIP calculator.
+        Requires the 'plumed' extra to be installed. Default is None.
```

Comment:
I think this should just be a path. (It can probably stay as `PathLike`, but `Path(plumed_input)` should exist).